src/data-types/wrappers.h

```diff
@@ -0,0 +1,326 @@
+#pragma once
+
+#include <errno.h>
+#include <stdarg.h>
+
+#ifdef EOF // stdio.h
+
+static inline FILE * Fopen(const char *filename, const char *mode)
+{
+    FILE * f;
+
+    do {
+        f = fopen(filename, mode);
+    } while (f == NULL && errno == EINTR);
+
+    return f;
+}
+#define fopen >@<
```

Comment:
I'm not sure to understand this syntax. Is it part of C standard?
What do you use that for since you changed all files that are making calls to fopen() to use Fopen()?